### DISCUSSION TITLE
onEvents calendar disapears

### DISCUSSION THREAD

**Discussion by ehs4nm** [saade/filament-fullcalendar#2] [Q&A] [answered]
hi. thanks for this great package.
When i use events like: onEventDrop, any updates make calender disapear and an empty div with intial js code goes there.

**Comment by saade** [comment]
Hey! Thanks for reporting that. I think is just a matter of adding `wire:ignore` to the calendar div.

Gonna fix that asap!

**Comment by saade** [comment, marked as answer]
Fixed in v0.2.1!
